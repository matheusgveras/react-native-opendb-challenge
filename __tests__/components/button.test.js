import 'react-native';
import React from 'react';
import Button  from '../../src/components/button';
import {Text} from 'react-native'
import renderer from 'react-test-renderer';
describe('testing button component', () => {
    
    it('renders buttom', () => {
        const view = <Button><Text>My button</Text></Button>
        const tree = renderer.create(view).toJSON();
        expect(tree).toMatchSnapshot();
    });

});