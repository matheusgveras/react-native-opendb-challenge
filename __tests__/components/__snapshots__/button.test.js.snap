// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing button component renders buttom 1`] = `
<View>
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "borderRadius": 5,
        "height": 42,
        "justifyContent": "center",
        "opacity": 1,
        "paddingHorizontal": 15,
      }
    }
  >
    <Text>
      My button
    </Text>
  </View>
</View>
`;
